src/extension.ts

```diff
@@ -134,7 +135,13 @@ export function activate(context: vscode.ExtensionContext): void {
         try {
             setSyncingStatusBarItem();
             const filePath = fileUri ? fileUri.fsPath : editor.document.fileName;
-            const checkovResponse = await runCheckovScan(logger, extensionVersion, filePath, token, cancelToken);
+            
+            if (!checkovInstallation) {
```

Comment:
We already have the `extensionReady` flag that indicates checkov has installed correctly, why do we need this variable too?
If you want to check also here that checkov is installed, use the existing variable.